sql/moz-fx-data-shared-prod/addons/addon_events_clients/view.sql

```diff
@@ -0,0 +1,37 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.addons.addon_events_clients`
+AS
+SELECT
+  submission_date,
+  `timestamp`,
+  COALESCE(
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addonId'),
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addon_id')
+  ) AS addon_id,
+  COALESCE(
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addonVersion'),
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addon_version')
+  ) AS addon_version,
+  -- TODO: why in some redash queries this is hashed_addon_id and in other value? What's the difference?
```

Comment:
This depends on what gets passed. The `hashed_addon_id` is only valid for `install_stats`: https://searchfox.org/mozilla-central/rev/5b061cdc4d40d44988dc61aa941cfbd98e31791f/toolkit/components/telemetry/Events.yaml#233-234